test/src/test/resources/fixtures/oneOf/one-of-fails-when-2-values-are-passed.yml

```diff
@@ -0,0 +1,38 @@
+name: "oneOf fails when 2 values are passed"
+enabled: true
+overallSchema:
+  MyService: |
+    type Query {
+      search(by: SearchInput): String
```

Comment:
You prob already had this in mind - when the new graphql-java build version containing your PR https://github.com/graphql-java/graphql-java/pull/3353 is merged in, can you add a test with the non-null input case?

Reply:
Yep, that's a good idea!